packages/edit-post/src/index.js

```diff
@@ -86,10 +86,10 @@ export function initializeEditor( id, postType, postId, settings, initialEdits )
 
 	const isIphone = window.navigator.userAgent.indexOf( 'iPhone' ) !== -1;
 	if ( isIphone ) {
-		document.addEventListener( 'focusin', function( ) {
-			setTimeout( () => {
+		window.addEventListener( 'scroll', function( event ) {
```

Comment:
So the purpose of this is to prevent scrolling on the window? Has the `overflow` technique been tried? (I don't see it mentioned in the comment.)

Reply:
The purpose is to compensate for the scrolling that Mobile Safari does due to a very hacky decision to scroll up the `html` element whenever the softkeyboard is showing. The hack is better described in https://github.com/WordPress/gutenberg/pull/18686 where the first version of this was merged.

Reply:
Assuming we're talking about the same technique, we did try the `overflow` technique while testing various approaches when we were working on https://github.com/WordPress/gutenberg/pull/18686. Unfortunately, most common approaches did not get us closer to fixing the issues we were seeing in mobile Safari as described in the previous PR as well as the related issue tickets.